### PR TITLE
fix: usage of `node12 which is deprecated`

### LINKED ISSUE #8790: `semantic-pr-title.yml` uses `action-semantic-pull-request@v3.0.0` which `uses node12 which is deprecated`
[`semantic-pr-title.yml` uses `action-semantic-pull-request@v3.0.0` which `uses node12 which is deprecated`](https://github.com/excalidraw/excalidraw/actions/runs/11759070409).

### PR DESCRIPTION
**Pull request by hamirmahal**
Fixes #8790

### CODE REVIEW

File: .github/workflows/semantic-pr-title.yml [hunk 1]
```diff
@@ -11,6 +11,6 @@ jobs:
   semantic:
     runs-on: ubuntu-latest
     steps:
-      - uses: amannn/action-semantic-pull-request@v3.0.0
+      - uses: amannn/action-semantic-pull-request@v5
```
Comment (hamirmahal) [inline .github/workflows/semantic-pr-title.yml:14]: `v5` is [the latest major version](https://github.com/amannn/action-semantic-pull-request/) at the time of this writing.

**Comment by hamirmahal** [comment]
# Without this change

image

**Comment by hamirmahal** [comment]
# With this change

image

File: .github/workflows/semantic-pr-title.yml (hunk 1 above)
Comment (hamirmahal) [inline .github/workflows/semantic-pr-title.yml:14]: `v5` is the latest major version of [`amannn/action-semantic-pull-request`](https://github.com/amannn/action-semantic-pull-request/) at the time of this writing.

**Comment by dwelle** [comment]
sure, thanks!

**Review by dwelle — APPROVED** [review]

**Pull request merged**